### PR TITLE
Formula to UnicodeMath conversion

### PR DESCRIPTION
**Pull request by suleman-uzair**
This PR adds conversion methods to convert **Formula** to **UnicodeMath**.

### CODE REVIEW

File: lib/plurimath/math/function/multiscript.rb [hunk 1]
```diff
@@ -78,6 +84,32 @@ def prescripts
...
+
+        def sub_value
+          field = Utility.filter_values(parameter_two)
+          if field&.mini_sized?
+            parameter_two.map(&:to_unicodemath).join
+          elsif parameter_two.is_a?(Math::Function::Base)
+            "_#{parameter_two.map(&:to_unicodemath).join}"
+          else
+            "_(#{parameter_two.map(&:to_unicodemath).join})" unless parameter_two.empty?
+          end
```
Comment (HassanAkbar) [inline lib/plurimath/math/function/multiscript.rb:111]:
```suggestion
          elsif parameter_two && !parameter_two.empty?
            "_(#{parameter_two.map(&:to_unicodemath).join})"
          end
```

File: lib/plurimath/math/function/nary.rb [hunk 2]
```diff
@@ -135,6 +145,41 @@ def omml_nary_tag(display_style)
...
+          else
+            "_#{unicodemath_parens(parameter_two)}"
+          end
+        end
+
+        def prime_unicode?(field)
+          return unless field.is_a?(Math::Symbol)
+
+          UnicodeMath::Constants::PREFIXED_PRIMES.any? { |prefix, prime| field.value.include?(prime) || field.value.include?("&#x27;") }
+        end
```
Comment (HassanAkbar) [inline lib/plurimath/math/function/nary.rb:175]:
```suggestion
        def prime_unicode?(field)
          return unless field.is_a?(Math::Symbol)
          return true if field.value.include?("&#x27;")

          UnicodeMath::Constants::PREFIXED_PRIMES.any? { |_prefix, prime| field.value.include?(prime) }
        end
```

File: lib/plurimath/math/function/ternary_function.rb [hunk 3]
```diff
@@ -180,6 +180,19 @@ def underover(display_style)
 
           Underset.new(overset, parameter_two)&.to_omml_without_math_tag(display_style)
         end
+
+        def naryand_value(field)
+          return "" unless field
+
+          field_value = field.to_unicodemath
+          field.is_a?(Math::Function::Fenced) ? "▒#{field_value}" :  "▒〖#{field_value}〗"
```
Comment (HassanAkbar) [inline lib/plurimath/math/function/ternary_function.rb:188]:
```suggestion
          field.is_a?(Math::Function::Fenced) ? "▒#{field_value}" : "▒〖#{field_value}〗"
```

**Review by HassanAkbar — APPROVED** [review]
@suleman-uzair overall the PR looks good. Just some minor code quality related comments.

File: lib/plurimath/math/function/multiscript.rb [hunk 4]
```diff
@@ -78,6 +84,32 @@ def prescripts
         def valid_value_exist?(field)
           !field && (field&.empty? || field&.all?(None))
         end
+
+        def unicode_valid_value?(field)
+          !(field.empty? && valid_value_exist?(field))
```
Comment (HassanAkbar) [inline lib/plurimath/math/function/multiscript.rb:89]: @suleman-uzair I think this condition will always return true, can you verify if this.

**Review by HassanAkbar — COMMENTED** [review]
@suleman-uzair  can you check this.

**Comment by ronaldtse** [comment]
@suleman-uzair is this ready to go? Thanks.

**Review by ronaldtse — APPROVED** [review]

**Pull request merged**